index.bs

```diff
@@ -493,7 +493,10 @@ This Section defines a number of new set of <a>Constrainable Properties</a> for
   <dd>This reflects the supported range of <a>contrast</a>. Values are numeric.  Increasing values indicate increasing contrast.</dd>
 
   <dt><dfn dict-member for="MediaTrackCapabilities"><code>pan</code></dfn></dt>
-  <dd>This reflects the <a>pan</a> value range supported by the UA.</dd>
+  <dd>This reflects the <a>pan</a> value range supported by the UA and by the track.
+
+  If the track has been created without <a>requesting permission to use</a> (as defined in [[!permissions]]) a PermissionDescriptor with its name member set to {{PermissionName/camera}} and its {{CameraDevicePermissionDescriptor/panTiltZoom}} member set to true or if that permission request is denied, the track does not support <a>pan</a>.
+  In that case the UA MUST NOT expose the <a>pan</a> value range.</dd>
```

Comment:
I would split this PR in two, one for getSettings and one for getCapabilities.
This change would disallow some step-by-step permission flow (first request camera permission, then request PTZ permission if PTZ support)

Reply:
I will two separate PRs.

The idea (and way it is currently implemented in Chrome) is that capabilities are guarded by the PTZ permission so that PTZ ranges cannot be used for fingerprinting without explicit permission grant.

Reply:
Let's continue at https://github.com/w3c/mediacapture-image/pull/250